code/game/objects/items/weapons/weaponry.dm

```diff
@@ -47,11 +47,11 @@
 	w_class = 3
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
-
+	
 /obj/item/weapon/sord/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is impaling \himself with the [src.name]! It looks like \he's trying to commit suicide.</span>")
-	return(BRUTELOSS)
-
+	user.visible_message("<span class='suicide'>You try to impale yourself with the SORD... and isn't even cut! Wow, that's a shitty sword...</span>")

```

Comment:
The first argument of visible_message() is what every viewers see, so you to use [user] instead of "You". Use the second argument to make a special message for the user.